test/1_stdlib/POSIX.swift

```diff
@@ -78,6 +81,112 @@ POSIXSemaphore.test("sem_open existing O_EXCL fail") {
   let res2 = sem_unlink(semaphoreName)
   expectEqual(0, res2)
 }
+	
+// Fail because file doesn't exist.
+POSIXTests.test("fcntl fail") {
+  let fd = open(strdup(fn), 0)
+  expectEqual(-1, fd)
+  expectEqual(ENOENT, errno)
+	
+  let _ = fcntl(fd, F_GETFL)
+  expectEqual(EBADF, errno)
+}
+
+// Change modes on existing file.
+POSIXTests.test("fcntl F_GETFL/F_SETFL success with file") {
+  // Create and open file.
+  system("touch \(fn)")
```

Comment:
Ok, will make the change...

> On Dec 10, 2015, at 1:40 PM, Dmitri Gribenko <notifications@github.com> wrote:
> 
> In test/1_stdlib/POSIX.swift <https://github.com/apple/swift/pull/196#discussion_r47265320>:
> 
> > @@ -78,6 +81,112 @@ POSIXSemaphore.test("sem_open existing O_EXCL fail") {
> >    let res2 = sem_unlink(semaphoreName)
> >    expectEqual(0, res2)
> >  }
> > +	
> > +// Fail because file doesn't exist.
> > +POSIXTests.test("fcntl fail") {
> > +  let fd = open(strdup(fn), 0)
> > +  expectEqual(-1, fd)
> > +  expectEqual(ENOENT, errno)
> > +	
> > +  let _ = fcntl(fd, F_GETFL)
> > +  expectEqual(EBADF, errno)
> > +}
> > +
> > +// Change modes on existing file.
> > +POSIXTests.test("fcntl F_GETFL/F_SETFL success with file") {
> > +  // Create and open file.
> > +  system("touch \(fn)")
> system() won't work on iOS, you need to create file manually.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/apple/swift/pull/196/files#r47265320>.
>